[W-15751811] Doc updates to client management pages (#338)
mulesoft/docs-access-management: modules/ROOT/pages/configure-client-management-openid-task.adoc

| Token Introspection | {BASE URL}/introspect | {BASE URL}/introspect | {BASE URL}/introspect
|===

> == How to Configure

Reviewer: ```suggestion
== Configuring Client Management

```

In *Advanced Settings* you can also select:

* *Disable server certificate validation*: Disables server certificate validation if your OpenID client management instance presents a self-signed certificate, or one signed by an internal certificate authority.
* *Enable client deletion in Anypoint Platform*: Enables deletion of clients created with this integration.
* *Enable client deletion and updates in IdP*: To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform to `{clientRegistrationUrl}/{clientID}`. The `clientRegistrationUrl` is the value you configured in *Client Registration URL*. For example: `+https://identity.example.com/oauth2/connect/register/6779ef20e75817b79602+`. For detailed example, please see below.

Once this is successfully configured, you can apply the OpenID Connect OAuth Token Enforcement policy to your API Gateways through API Manager. Requesting API access through API portals dynamically generates client applications in the configured IDP that acts as a token provider.

[NOTE]

> For Okta, the Okta admin needs to assign the dynamically generated clients to a user or a group of users in order for them to receive access tokens by sending over the Client ID and Client Secret.

Reviewer: I wouldn't put two notes in a row - is this something that can be moved higher in the article?

Reviewer: Or combined somewhere

Author: Good question.. I just copied the previous docs and they were together. Let me try moving one.

[NOTE]

> When Anypoint Platform makes an HTTP call to the configured OIDC endpoints, the HTTP call times out if no response after 5 seconds.

Reviewer: ```suggestion
Anypoint Platform makes an HTTP call to the configured OIDC endpoints, and if there is no response after five seconds, the HTTP call times out.
```

== Client Application Updates and Deletes

> Anypoint platform embraces OpenID Connect Dynamic Client Registration, and hence client modification or deletion is not supported, although OKTA or another IDP might do so independently. But, as mentioned above we instead provide and support hooks for someone to implement API calls that are particular to their IDP. Below are samples for PUT and DELETE requests Anypoint will make.

Reviewer: ```suggestion
Anypoint Platform embraces OpenID Connect Dynamic Client Registration. As a result, client modification and deletion are not supported, although OKTA or another IDP may allow it independently. Instead, Anypoint Platform provides and supports hooks so you can implement API calls that are particular to your IDP. The following examples show PUT and DELETE requests that Anypoint Platform will make.
```

     "client_name": "test-client-name"
}
----
* The `Authorization:` header is included as part of the request only if the `Authorization Header` under *Advanced settings* is configured.
* The `client_id` request parameter that is passed is the same as the `client_id` passed in the `PUT` request payload.
* The `token_endpoint_auth_method` passed in the payload is always `client_secret_basic`.
* Only the `Client Registration URL` and `Authorization Header` come from what is configured in OIDC *Client Provider*.

+
[NOTE]
`client_name` in the request maps to `name` on the inbound side. +
The authorization method depends on the grant type. For example, if the grant type is `implicit`, the `id_token` and `token` are returned.

> This is another example of the DELETE request header:

Reviewer: ```suggestion
The following example shows another DELETE request header:
```
